### ISSUE TITLE
Not visible on datasource manager with QGIS 3.24

### ISSUE THREAD

**Issue by Doctor-Who** [camptocamp/QGIS-SpreadSheetLayers#28]
Hello,

 _ | _
-- | -- 
Version de QGIS | 3.24.0-Tisler 
Révision du code | 6b44a42058
Version de Qt | 5.15.2
Version de Python | 3.9.7
Version de GDAL/OGR | 3.2.2
Version de Proj | 7.2.1
Version de la base de données du registre EPSG | v10.008 (2020-12-16)
Version de GEOS | 3.9.0-CAPI-1.16.2
Version de SQLite | 3.35.5
Version de PDAL | 2.2.0
Version du client PostgreSQL | 13.5 (Ubuntu 13.5-0ubuntu0.21.10.1)
Version de SpatiaLite | 5.0.1
Version de QWT | 6.1.4
Version de QScintilla2 | 2.11.6
Version de l'OS | Pop!_OS 21.10

Not visible anymore on left panel from the datasource manager : 

![image](https://user-images.githubusercontent.com/5890416/159241937-76aa0bc8-d067-4578-9fb5-d98f6dc42fbf.png)

Still available from top text menu > layers > add a layer

**Comment by effjot** [comment]
It’s not in the Data Source Manager in version 3.22.4 either, and also not in the Browser panel. However, I’m not sure if I’ve ever seen it there.

**Comment by Doctor-Who** [comment]
@effjot you're totally right .... My bad, I've check on LTR and it was located on the layer toolbar not on the new data source manager.
However it could be consider as an enhancement request :)
